nilearn/plotting/img_plotting.py

```diff
@@ -197,7 +197,7 @@ def _plot_img_with_bg(
     %(display_mode)s
 
     %(colorbar)s
-        Default=False.
+        Default=True.
```

Comment:
```suggestion
        Default=False.
```

revert on private function